### DISCUSSION TITLE
Is it possilble to use other models (not in Models Registry)

### DISCUSSION THREAD

**Discussion by gilgamsh** [TabbyML/tabby#1353] [Ideas]
I have read https://github.com/TabbyML/tabby/issues/1230 this issue #1230 . 
But I want to know if huggingface does not have the gguf format. How should I do? 
(I want to try other models better than [TabbyML/DeepseekCoder-6.7B](https://huggingface.co/deepseek-ai/deepseek-coder-6.7b-base), since DeepseekCoder-6.7B is still not the limit of my device)

**Comment by wsxiaoys** [comment]
Yes, please refer "How can I convert my own model for use with Tabby?" in https://tabby.tabbyml.com/docs/faq/

> But I want to know if huggingface does not have the gguf format. How should I do?

You could either convert it by your self, or see if https://huggingface.co/TheBloke has converted it for you, for example https://huggingface.co/TheBloke/CodeLlama-70B-hf-GGUF/tree/main

**Comment by gilgamsh** [comment]
Thank  you very much.
